content/messaging/customer-chat-api/v3.2/rtm-reference/index.mdx

```diff
@@ -124,59 +142,38 @@ File event informs about a file being uploaded.
-| `fields.answer.id`      | N/A        | `string`                | For `email`, `name, `radio`, `group_chooser`                                                         |
-| `fields.answer.label`   | required   | `string`                | For `email`, `name, `radio`, `group_chooser`                                                         |
-| `fields.answer.group_id`| required   | `number`                | For `group_chooser`                                                                                      |
-| `fields.answers`        | required   | `array of objects`      | For `checkbox`                                                                                           |
-| `fields.answers.id`     | required   | `string`                | For `checkbox`                                                                                           |
-| `fields.answers.label`  | required   | `string`                | For `checkbox`                                                                                           |
+| Field                   | Required | Data type  |  Notes                                                                    |
+| ------------------------| -------- | -----------| ------------------------------------------------------------------------- |
+| `custom_id`             | no       | `string`   |                                                                           |
+| `type`                  | yes      | `string`   | `filled_form`                                                             |
+| `recipients`            | yes      | `string`   | Possible values: `all` (default), `agents`                                |
+| `properties`            | no       | `object`   | [Properties](#properties)                                                 |
+| `form_id`               | ?        | `string`   |                                                                          |
+| `fields`                | yes      | `array`    | The fields a form contains. See [form fields](#form-fields) for details. |
 
 ### Responses
 
 | Field                   | Returned    | Notes |
 | ------------------------| ----------- | ----- |
 | `id`                    | always      | Generated server-side |
 | `custom_id`             | optionally  | |
-| `created_at`            | always      | Generated server-side; Date & time format with a resolution of microseconds, `UTC string`. |
+| `created_at`            | always      | Generated server-side; date & time format with a resolution of microseconds, `UTC string`. |
 | `type`                  | always      | |
 | `author_id`             | always      | Generated server-side |
 | `recipients`            | always      |  |
 | `properties`            | optionally  |  |
 | `form_id`               | ?           |  |
 | `fields`                | always      |  |
```

Comment:
shouldn't there be a note that `fields` is array of `form fields` objects?

Reply:
@litwinow 
what about `form_id` in requests? is it optional or required?

Reply:
it's required